integration/kubernetes/k8s-caps.bats

```diff
@@ -33,6 +33,12 @@ setup() {
         kubectl create -f "${pod_config_dir}/pod-caps.yaml"
         # Check pod creation
         kubectl wait --for=condition=Ready --timeout=$timeout pod "$pod_name"
+
+        # Small wait to ensure the container completed execution and has the expected logs
+        # available:
+        sleep 1
```

Comment:
Hi @egernst , one second might still be not sufficient... and we will need to increase it... what if you use `waitForProcess` with a sleep time of 1 second and wait time of, say, 10 seconds?

Reply:
I see.  In my case, I'm not waiting for a particular process to complete, since the container itself will keep on executing after it'll output logs. 

I agree, sprinkling "sleep" in a test is generally horrible. In this case, its a pretty infrequent flake - I was surprised to see the container not start executing immediately once the sandbox is marked ready.

I guess a solution could be to check the logs, and retry after a second if we don't see any of the expected log output?